backend/EasySpeak.Core/EasySpeak.Core.Common/DTO/Rabbit/RecommendationServiceMessageDto.cs

```diff
@@ -0,0 +1,14 @@
+using EasySpeak.Core.Common.Converters;
+using EasySpeak.Core.Common.Enums;
+using Newtonsoft.Json;
+
+namespace EasySpeak.Core.Common.DTO.Rabbit;
+
+public class RecommendationServiceMessageDto
+{
+    public QueryType Type { get; set; }
+    public Dictionary<string, object> Parameters { get; set; } = new();
+    
+    [JsonConverter(typeof(ListStringConverter))]
+    public List<string>? ParameterList { get; set; } = new();
```

Comment:
Use string[], it should work without converters

Reply:
I've checked how it works with array, and result is the same. Anyway, I will check it again, and if it works fine, I'll change it